src/js/components/AppBody.vue

```diff
@@ -8,14 +8,14 @@
     <main-menu
       :open="menuOpen"
       :keepOpen="keepMenuOpen"
-      :corpus="menuCorpus"
+      :corpus="corpus"
       @mouseenter='mouseOver = true'
       @mouseleave='mouseOver = false'
     />
 
     <!-- editing panes -->
     <div id="editing-window" :class='{"open": menuOpen}'>
-      <header-menu></header-menu>
+      <header-menu :corpus="corpus"></header-menu>
```

Comment:
This seems like a nice approach to me so that all of the panels know what's going on.

Reply:
Thanks, I it certainly works.  I also sort of wondered about having this object in Vue store or in some more centralized spot.  The idea being that this whole structure is in a way stateful, but it is also potentially huge.  If you are happy passing it from component to component is good, then we can keep it this way.